server.js

```diff
@@ -10,7 +10,7 @@ const connectDB = require('./config/database')
 const mainRoutes = require('./routes/main')
 const todoRoutes = require('./routes/todos')
 
-require('dotenv').config({path: './config/.env'})
```

Comment:
the code is looking for the `.env` file in the wrong location. By default the `dotenv` package looks for a `.env` file in the top level directory of the repository.